integr8ly/integreatly-operator
# Add OO to container images, products.yaml and some other fixes to prep/release

# Issue link
https://issues.redhat.com/browse/MGDAPI-2750

# What
1. Fix issue with container images generation where due to new manifest for kc and 3scale we had some duplication in images and some empty URLs
2. Add manifest for OO (although it is installed via index we need manifest to keep some of our pipelines happy)
3. Re-generate master manifests and adding OO to products.

# Verification steps
Review file changes & run `OLM_TYPE=managed-api-service SEMVER=1.14.0 make release/prepare` and confirm CSV for new version of RHOAM generates successfully. No install necessary.
